src/Transform/BundleMemoryPools.cpp

```diff
@@ -320,8 +320,7 @@ class KrnlBundleDynamicMemoryPools : public OpRewritePattern<AllocOp> {
             // Check if the current operation is a DimOp or a LoadOp.
             if (llvm::dyn_cast<DimOp>(definingOperation) ||
                 llvm::dyn_cast<KrnlDimOp>(definingOperation) ||
-                llvm::dyn_cast<LoadOp>(definingOperation) ||
-                llvm::dyn_cast<AffineLoadOp>(definingOperation)) {
+                llvm::dyn_cast<KrnlLoadOp>(definingOperation)) {
```

Comment:
When is the lowering of krnl.load/store performed?

Reply:
When we lower krnl to affine, i.e. inside convert-krnl-to-affine pass.